tds/routers/datasets.py

```diff
@@ -53,8 +53,9 @@ def create_feature(payload: schema.Feature, rdb: Engine = Depends(request_rdb)):
         featurep = payload.dict()
         del featurep["id"]
         feature = orm.Feature(**featurep)
-        exists = session.query(orm.Feature).filter_by(**featurep).first() is not None
-        if exists:
+        exists = session.query(orm.Feature).filter_by(**featurep).first()
+        if exists is not None:
+            featurep["id"] = exists.id
```

Comment:
Let's rename exists to `feature_result` or something equivalent if this isn't used outside this block.
As we change the usage of a variable, we should also change the name of the variable.

Reply:
Changed